scripts/metadata_comparison/metadata_comparison/comparer.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+#
+# comparer.py
+#
+# Purpose: Compare performance metadata JSON files produced by Digester and produce result in CSV format
+#
+# Usage: python3 comparer.py [-h] [-v] [--json_paths JSONPATH [JSONPATH ...]] [--output_path OUTPUTPATH]
+#
+# Python Prereqs (at least, the ones which I needed to manually install... YMMV):
+#
+#   * pip3 install --upgrade pandas
+#
+
+import argparse
+import json
+import pandas
+from pathlib import Path
+import logging as log
+
+logger = log.getLogger('metadata_comparison.comparer')
+
+def set_log_verbosity(verbose):
+    if verbose:
+        log.basicConfig(format='[%(asctime)s] [%(name)s] %(message)s', level=log.INFO)
+    else:
+        log.basicConfig(format='[%(asctime)s] [%(name)s] %(message)s', level=log.WARNING)
+
+
+def read_json_files(*paths):
+    """
+    Produces list of tuples: [[path, json], [path, json], ...]
+    """
+    jsons = []
+    for path in paths:
+        jsons.append([path, json.load(open(path,))])
+
+    return jsons
+
+def compare_jsons(*pathsAndJsons):
+    """
+    Uses pandas library to convert JSONs into into dataframes, and concatenate those dataframes into a single one.
+    Performs sanity check, producing exception, if at least one of the JSONs doesn't have matching subset of keys.
```

Comment:
This looks like something that could have a unit test or two.

As well as making sure it's correct and stopping regressions (both good reasons!), it'd also help me to make sense of what the expected output for the function is for a given input!

Reply:
speaking of unit tests it would be nice if they were being run in CI so broken tests/code didn't get checked in... 😬